src/components/Navbar/Navbar.jsx

```diff
@@ -0,0 +1,27 @@
+import { Link } from "react-router-dom";
+import { Flex, Spacer, Box, Image, Avatar } from "@chakra-ui/react";
+import "../../styles.css";
+import Logo from "../../Assets/neoG.camp.svg";
+
+export function Navbar() {
+  return (
+    <div>
+      <Flex width="100%">
+        <Box p="5" pl="10rem">
+          <Link to="/">
+            <Image src={Logo} />
```

Comment:
Use `alt` attribute to image element
```suggestion
            <Image src={Logo} alt='neogcamp logo' />
```

Reply:
Yeah done, Thanks for the suggestion.